### PR TITLE
testing: rollout 31.20200108.2.0

### PR DESCRIPTION
**Pull request by dustymabe**

### CODE REVIEW

File: updates/testing.json [hunk 1]
```diff
@@ -21,18 +21,18 @@
...
           "start_percentage": 1.0
         }
       }
     },
     {
-      "version": "31.20191217.2.0",
+      "version": "31.20200108.2.0",
       "metadata": {
         "rollout": {
-          "start_epoch": 1576765800,
```
Comment (dustymabe) [inline updates/testing.json]: i'm hesitant on this. I hate to say it but should we wait til monday?

Comment (jlebon) [reply updates/testing.json]: Yeah, I think that makes sense.

Comment (dustymabe) [reply updates/testing.json]: ok fixed

Comment (jlebon) [reply updates/testing.json]: Monday 9AM. :heavy_check_mark:

Comment (dustymabe) [reply updates/testing.json]: EST, yes :)

**Pull request merged**